src/test/java/org/folio/service/export/ExportServiceUnitTest.java

```diff
@@ -115,6 +116,23 @@ void shouldSaveErrorLog_whenJsonCannotBeConvertedToMarcRecordBecauseOfLargeSize(
09406\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"035\":{\"subfields\":[{\"a\":\"(OCoLC)32923445\"},{\"z\":\"(OCoLC)36260797\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"035\":{\"subfields\":[{\"a\":\"(OCoLC)ocm32923445\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"040\":{\"subfields\":[{\"a\":\"DLC\"},{\"b\":\"eng\"},{\"c\":\"DLC\"},{\"d\":\"UKM\"},{\"d\":\"BAKER\"},{\"d\":\"BTCTA\"},{\"d\":\"YDXCP\"},{\"d\":\"DEBBG\"},{\"d\":\"OCL\"},{\"d\":\"BDX\"},{\"d\":\"OCLCF\"},{\"d\":\"OCLCQ\"},{\"d\":\"OCLCO\"},{\"d\":\"TXM\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"041\":{\"subfields\":[{\"a\":\"rus\"},{\"b\":\"eng\"}],\"ind1\":\"0\",\"ind2\":\" \"}},{\"049\":{\"subfields\":[{\"a\":\"TXMM\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"050\":{\"subfields\":[{\"a\":\"PG2117\"},{\"b\":\".G59 1996\"}],\"ind1\":\"0\",\"ind2\":\"0\"}},{\"066\":{\"subfields\":[{\"c\":\"(N\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"082\":{\"subfields\":[{\"a\":\"491.786/421\"},{\"2\":\"20\"}],\"ind1\":\"0\",\"ind2\":\"0\"}},{\"092\":{\"subfields\":[{\"a\":\"491.786\"},{\"b\":\"G5651\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"245\":{\"subfields\":[{\"a\":\"The Golden Age :\"},{\"b\":\"readings in Russian literature of the nineteenth century /\"},{\"c\":\"[compiled by] Sandra F. Rosengrant, Elena D. Lifschitz.\"}],\"ind1\":\"0\",\"ind2\":\"4\"}},{\"260\":{\"subfields\":[{\"a\":\"New York :\"},{\"b\":\"J. Wiley,\"},{\"c\":\"©1996.\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"300\":{\"subfields\":[{\"a\":\"v, 375 p. :\"},{\"b\":\"ill. ;\"},{\"c\":\"24 cm.\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"500\":{\"subfields\":[{\"a\":\"UTF8  leader encoded with UTF8 record\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"504\":{\"subfields\":[{\"a\":\"Includes bibliographical references.\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"650\":{\"subfields\":[{\"a\":\"Russian language\"},{\"v\":\"Readers.\"}],\"ind1\":\" \",\"ind2\":\"0\"}},{\"650\":{\"subfields\":[{\"a\":\"Russian literature\"},{\"y\":\"19th century.\"}],\"ind1\":\" \",\"ind2\":\"0\"}},{\"923\":{\"subfields\":[{\"g\":\"very_long_volume very_long_volume very_long_volume very_long_volume very_long_volume very_long_volume very_long_volume \"},{\"3\":\"ho00000000002\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"945\":{\"subfields\":[{\"a\":\"491.786\"},{\"b\":\"G5651\"},{\"g\":\"1\"},{\"i\":\"33082017611558\"},{\"j\":\"0\"},{\"l\":\"w3   \"},{\"o\":\"-\"},{\"p\":\"$76.68\"},{\"q\":\"-\"},{\"r\":\"-\"},{\"s\":\"-  \"},{\"t\":\"0\"},{\"u\":\"3\"},{\"v\":\"0\"},{\"w\":\"0\"},{\"x\":\"3\"},{\"y\":\".i21033298\"},{\"z\":\"05-22-15\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"994\":{\"subfields\":[{\"a\":\"C0\"},{\"b\":\"TXM\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"998\":{\"subfields\":[{\"a\":\"w\"},{\"b\":\"05-26-15\"},{\"c\":\"m\"},{\"d\":\"a  \"},{\"e\":\"-\"},{\"f\":\"eng\"},{\"g\":\"nyu\"},{\"h\":\"4\"},{\"i\":\"1\"}],\"ind1\":\" \",\"ind2\":\" \"}},{\"999\":{\"subfields\":[{\"s\":\"b26859f1-8c79-47b3-b047-1ecde668492f\"},{\"i\":\"6666df22-5df3-412b-b9cc-cbdddb928e93\"}],\"ind1\":\"f\",\"ind2\":\"f\"}}]}\n";
```

Comment:
String record looks a quite large, maybe is it better to read it from some source for tests.

Reply:
I tried, but in this case all control characters in the obtained string have been already escaped.

Reply:
I reduced test record.